courses/free-courses-en.md

```diff
@@ -733,6 +733,7 @@
 * [Graph Theory](https://youtube.com/playlist?list=PLztBpqftvzxXBhbYxoaZJmnZF6AUQr1mH) - Wrath of Math
 * [Graph Theory - DM](https://youtube.com/playlist?list=PLYrahs7hsYIQiSNxTfZndQz7jWPXsA1ur) - SCCI Labs IIT Ropar
 * [Graph Theory Playlist](https://youtube.com/playlist?list=PLDV1Zeh2NRsDGO4--qE8yH72HFL1Km93P) - WilliamFiset
+* [Graph Theory Playlist](https://www.youtube.com/playlist?list=PLC36xJgs4dxHYyeppopMuALQhKY0ADIHi) - GATE CSE LECTURES BY AMIT KHURANA
```

Comment:
Use the same title from the provided playlist link and keep the link in a single line.